### DISCUSSION TITLE
Login to vercel is not working

### DISCUSSION THREAD

**Discussion by VenV-developers** [vercel/community#3317] [Help] [answered]
### Summary

I am trying to log in to vercel from a week but it is not getting me logged in just showing me errors
<img width="324" alt="image" src="https://github.com/vercel/community/assets/139146386/24c28bfa-9889-400b-a569-edd161aee5cd">
<img width="340" alt="image" src="https://github.com/vercel/community/assets/139146386/28630514-12a5-4d91-8d0a-0490b79193c5">

### Steps to Reproduce

login is not working or my account is blocked please help me anyone in whitelisting it.

**Comment by amyegan** [comment, marked as answer]
Hey @VenV-developers. Your login problem seems related to this incident https://www.vercel-status.com/incidents/tp22kxv7rfcm
